### DISCUSSION TITLE
单引号括号的问题

### DISCUSSION THREAD

**Discussion by SandwichBK** [cpeditor/cpeditor-cn#21] [提问] [answered]
![image](https://user-images.githubusercontent.com/53464864/183279357-87e85a1c-0f87-435b-a548-3fd52419f3b2.png)
如上图，单引号的括号不会被高亮，似乎无法被识别
同时，出现了如下图所示的错误的括号匹配
![image](https://user-images.githubusercontent.com/53464864/183278943-a3f581ad-57f3-4b46-99f6-7b47ee77533b.png)

**Comment by ouuan** [comment, marked as answer]
是已知问题 https://github.com/cpeditor/cpeditor/issues/174
